client/extensions/woocommerce/app/dashboard/manage-no-orders.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import BasicWidget from '../../components/basic-widget';
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc